web/src/components/organisms/Asset/AssetList/assetSortType.type.ts

```diff
@@ -0,0 +1,14 @@
+import { AssetSortType as GQLSortType } from "@reearth-cms/gql/graphql-client-api";
+
+export type AssetSortType = "date" | "name" | "size";
+
+const enumTypeMapper: Partial<Record<GQLSortType, string>> = {
+  [GQLSortType.Date]: "date",
+  [GQLSortType.Name]: "name",
+  [GQLSortType.Size]: "size",
+};
+
+export function toGQLEnum(val?: AssetSortType) {
```

Comment:
Is this function necessary?
Maybe type assertion (using as) is enough when needed like this:
`sort: sort?.type as GQLSortType,`

Reply:
Done!